code/API_definitions/quality-on-demand.yaml

```diff
@@ -840,12 +840,21 @@ components:
       minProperties: 1
 
     NetworkAccessIdentifier:
-      description: A public identifier addressing a subscription in a mobile network. In 3GPP terminology, it corresponds to the GPSI formatted with the External Identifier ({Local Identifier}@{Domain Identifier}). Unlike the telephone number, the network access identifier is not subjected to portability ruling in force, and is individually managed by each operator.
+      description: |
+        A public identifier for a mobile network subscription. In simple
+        terms, it follows the format "Local Identifier@Domain Identifier".
+        Unlike a telephone number, this identifier is fixed to one operator
+        and cannot be transferred between providers.
```

Comment:
Same as above ... we should first introduce a change to Commonalities

Reply:
reverted this change